account_invoice_qr_code_sepa_payconiq/models/res_partner_bank.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2022 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import base64
+import io
+import urllib
+
+import requests
+from PIL import Image
+
+from odoo import _, api, models
+from odoo.tools.image import image_data_uri
+
+PAYCONIQ_URL = "https://payconiq.com/t/1/"
+PAYCONIQ_QR_URL = "https://portal.payconiq.com/qrcode"
```

Comment:
LGTM

Just a small suggestion: It would be better to fetch this from ir.config_parameter, allowing admin to easily update the URL if Payconiq changes it in the future.